lib/absinthe/phase/telemetry.ex

```diff
@@ -10,12 +10,21 @@ defmodule Absinthe.Phase.Telemetry do
 
   use Absinthe.Phase
 
-  def run(blueprint, [:execute, :operation, :start]) do
+  def run(blueprint, options) do
+    event = Keyword.fetch!(options, :event)
+    do_run(blueprint, event, options)
+  end
+
+  defp do_run(blueprint, [:execute, :operation, :start], options) do
     id = :erlang.unique_integer()
     start_time = System.system_time()
     start_time_mono = System.monotonic_time()
 
-    :telemetry.execute(@operation_start, %{start_time: start_time}, %{id: id})
+    :telemetry.execute(@operation_start, %{start_time: start_time}, %{
+      id: id,
+      blueprint: blueprint,
+      options: options
```

Comment:
@binaryseed can you think of a reason to not push out the blueprint and options on all of these?

Reply:
probably good to be consistent.  the only worry is the size of the struct, hopefully no handler passes everything across a process boundary